### ISSUE TITLE
Properties conversion in the Redis-set response

### ISSUE THREAD

**Issue by Guttz** [RedisGraph/redisgraph-py#42]
When converting the properties with the new redis-set format, I'm getting properties strings with the byte annotation inside the dict keys and values instead of decoded. 

![image](https://user-images.githubusercontent.com/11739405/60533583-57b62500-9cd6-11e9-8221-fe280c864d9d.png)

Investigating a bit I found(query_result.py : 125):
![image](https://user-images.githubusercontent.com/11739405/60532503-cb0a6780-9cd3-11e9-83e0-05ab6c9da207.png)
 
Is this the expected or should the above be value.decode() instead of casting to string?

**Comment by Guttz** [comment]
Also, in the second property above, the value of "operations" has both` "` and` ' `switched which makes a bit harder to use some string manipulation.

**Comment by swilly22** [comment]
Hi @Guttz,
When setting up your Redis connection try specifying: `r=Redis(decode_responses=True)`

Please let me know if that helped.

**Comment by Guttz** [comment]
Hey @swilly22, it would make sanse for that to solve the problem. However, when doing that the following happens:

**Comment by Guttz** [comment]
Hey @swilly22, it would make sense for that to solve the problem. However, when doing that the following happens:

```
  File "/home/guttz/Documents/hydraeco/hydra-python-agent/venv/lib/python3.6/site-packages/redisgraph/query_result.py", line 57, in _get_value
    stat = stat.decode()
AttributeError: 'str' object has no attribute 'decode'
```

That happens because all statistics come as str already. Did a quick fix to see if that's the only problem:

![image](https://user-images.githubusercontent.com/11739405/60592730-9ea82700-9d77-11e9-8bf0-e53d298b5876.png)

With that, no error is thrown but the result-set conversion somehow fails and I get and unparsed result_set as the img:

![image](https://user-images.githubusercontent.com/11739405/60592770-b1226080-9d77-11e9-91a3-dc33f41f931e.png)

**Comment by jeffreylovitz** [comment]
Hi @Guttz,

It looks like the `redisgraph-py` package installed in your 3.6 environment is an older version, which won't properly handle the new result format. (The error reported on line 57 refers to code that was replaced in #28).

Can you try replacing the older package with one built from source? From the `redisgraph-py` root, this will require some command like `pip install --user .`

If this doesn't resolve the issue, it would be great if you could provide a code snippet to reproduce the issue!

**Comment by Guttz** [comment]
Hey @jeffreylovitz, you're right. I forgot to rebuild after checking out the branch to test @swilly22's solution. With his suggestion, it's working now. Thanks!

One last question, when running "MATCH(p) RETURN p.members", for example, **I will not get a result_set**:
`{"members" : "a,b,c.."}`

Instead, I get straight up the string with the value
`"a,b,c.."`

So, every time I query for properties, and not nodes, I should expect the return with no dict in the response, right?

**Comment by jeffreylovitz** [comment]
@Guttz Awesome, glad that worked!

Any query that returns values produces a result set, which is essentially a table that has columns, in order, corresponding to the entities or properties specified in your RETURN clause. A query like:
`"MATCH ()-[e]->(p) RETURN e, p.members"`
Will have the relationship `e` at at the first offset into the result set and the scalar value corresponding to the `members` property at the second.

The Python client performs a fair bit of local caching on things like property keys to reduce the package size for returning full nodes or relationships, but in general I would expect returning scalar values to be more performant when possible.

**Comment by Guttz** [comment]
Oh, I understand better now. Thanks a lot for the clarifications @jeffreylovitz. 

The issue was solved, I'll keep an eye for updates and the 2.0 release.
